src/data/nat/digits.lean

```diff
@@ -281,6 +236,56 @@ begin
   { simp [of_digits, list.sum_cons, ih], }
 end
 
+/-! ### Inequalities
+
+This section contains various lemmas with inequalities relating to `digits` and `of_digits`.
+ -/
+
+/-- The digits in the base b+2 expansion of n are all less than b+2 -/
+lemma digits_lt_base' {b m : ℕ} : ∀ {d}, d ∈ digits (b+2) m → d < b+2 :=
```

Comment:
I moved these down from just under digits to here, to be grouped with the rest of the inequalities. To be honest I don't have strong feelings about this, it's an inequality, but it is also a property of `digits`, which means it kind of fits in either section. Dependencies aren't an issue at either location.